src/foam/nanos/ticket/services.jrl

```diff
@@ -37,7 +37,8 @@ p({
       .setDecorator(
         new foam.dao.history.HistoryDAO(x, "ticketHistoryDAO",
           new foam.nanos.ticket.TicketAddCommentDAO.Builder(x).setDelegate(
-            new foam.nanos.ticket.TicketOwnerDAO.Builder(x).build()).build())
+            new foam.nanos.ticket.TicketOwnerDAO.Builder(x).setDelegate(
+              new foam.dao.NullDAO.Builder(x).setOf(foam.nanos.ticket.Ticket.getOwnClassInfo()).build()).build()).build())
```

Comment:
just curious, what is the effect of having a NullDAO in the decorator here?

Reply:
On this branch all EasyDAO decorators must be terminate by a NullDAO rather than null.  EasyDAO initialization fails with NPE if any ProxyDAO.init is called which access delegate.of.  With null, the last delegate is null, with NullDAO a valid of is available.